### ISSUE TITLE
create an app skeleton

### ISSUE THREAD

**Issue by shaharsol** [shaharsol/cmprop1#1]
nodejs express

codemill$20

**Comment by alexbowers** [comment]
For this to show on Codemill, I believe it has to be added as a label.

**Comment by shaharsol** [comment]
@alexbowers actually it's exactly as it supposed to be. I am the developer of CodeMill... this was just a sanity test when I took to the project to production ;-)

**Comment by alexbowers** [comment]
When I looked on codemill nothing was listed in 'open'

**Comment by shaharsol** [comment]
Tasks were grabbed like hot buns the last 24h. I think there is a single one now. Perhaps you were looking via the clients perspective? Try the [developers dashboard](http://codemill.io/developers/tasks/open)

**Comment by alexbowers** [comment]
I was definitely in the developer dashboard (and tried both). I thought
once a task was taken that there was a comment left on the issue with the
developers name

**Comment by shaharsol** [comment]
You're right. [Here's a live example](https://github.com/FedericoElles/codemill/issues/1)

**Comment by alexbowers** [comment]
So what I dont understand is why this one didn't show in 'Open' if it hasn't been assigned to anyone.

**Comment by shaharsol** [comment]
I assume that when you had a look it either didn't exist yet, or we had several tasks in the last 24h that were grabbed, and then the developers left them and they were back in the market, then grabbed again.

There's always the possibility of a bug however I didn't find anything suspicious in the logs.
